apps/api/src/app/auth/services/auth.service.ts

```diff
@@ -335,4 +337,31 @@ export class AuthService {
   private async getSubscriber({ subscriberId, _environmentId }: { subscriberId: string; _environmentId: string }) {
     return await this.subscriberRepository.findBySubscriberId(_environmentId, subscriberId);
   }
+
+  @CachedEntity({
```

Comment:
@djabarovgeorge what is our current default TTL?

Reply:
@scopsy I kept it as it is now (environment and user), it is 2 hours.

Reply:
We should clear it when the user `email` has changed, like in this use-case `UpdateProfileEmail`. Otherwise, the signed token might have an old email.